WordPress/Classes/Utility/Ratings/AppRatingUtility.m

```diff
@@ -319,4 +328,14 @@ + (void)assertValidSection:(NSString *)section
     NSAssert([[AppRatingUtility sharedInstance].sections.allKeys containsObject:section], @"Invalid section");
 }
 
++ (NSString *)appReviewUrl
+{
+    AppRatingUtility *sharedInstance = [AppRatingUtility sharedInstance];
+    if (sharedInstance.appReviewUrl.length == 0) {
+        return [NSString stringWithFormat:@"http://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=%@&pageNumber=0&sortOrdering=2&type=Purple+Software&mt=8", WPiTunesAppId];
```

Comment:
I'll just adjust this and drop the format stuff since it's not likely we're going to be changing App Id's